jsapp/js/account/organization/organizationQuery.ts

```diff
@@ -107,8 +106,7 @@ export const useOrganizationQuery = (options?: Omit<UndefinedInitialDataOptions<
   // the session data loaded. Account data is needed to fetch the organization
   // data.
   const isQueryEnabled =
-    !sessionStore.isPending &&
-    sessionStore.isInitialLoadComplete &&
+    !session.isPending &&
```

Comment:
I believe that using only the `!!organizationUrl` here is enough, since it'll only exist if the currentLoggedAccount is loaded.